tools/toolchain/dbuild

```diff
@@ -90,11 +91,19 @@ elif [[ "$1" = -* ]]; then
                 if [[ -z "$image" ]]; then
                     die "Expected docker image identifier to follow the --image option"
                 fi
-                if ! docker image inspect "$image" >/dev/null && ! docker image pull "$image"; then
+                if ! $tool image inspect "$image" >/dev/null && ! $tool image pull "$image"; then
                     die
                 fi
                 continue
                 ;;
+            --tool)
+                tool="$2"
+                shift 2
+                if [[ -z "$tool" ]]; then
+                  die "Expected tool name to use, available options: docker or podman"
```

Comment:
You say that the available options are just docker or podman, but this is not what the if tests. If someone does "dbuild --tool --xyz" this code will happy set tool="-xyz" :-)

Reply:
@nyh Removed this part, and the `--tool` flag